src/main/java/teammates/ui/webapi/action/SearchAccountsAction.java

```diff
@@ -16,7 +18,8 @@
 import teammates.common.util.Config;
 import teammates.common.util.Const;
 import teammates.common.util.StringHelper;
-import teammates.ui.webapi.output.ApiOutput;
+import teammates.ui.webapi.output.AdminSearchResultData;
+
 
 /**
  * Action: searches for accounts.
```

Comment:
Don't forget to update the javadoc

Reply:
What do you mean here?

Reply:
For most of your javadoc changes, simply removing the "Action:" part is sufficient. You can take a look at the completed ones (SaveFeedbackQuestionAction, CreateFeedbackQuestionAction, DeleteFeedbackQuestionAction etc.) for samples.